cmdstanpy/cmdstan_args.py

```diff
@@ -104,139 +114,199 @@ def validate(self, chains: Optional[int]) -> None:
         if self.iter_warmup is not None:
             if self.iter_warmup < 0 or not isinstance(self.iter_warmup, int):
                 raise ValueError(
-                    'iter_warmup must be a non-negative integer,'
-                    ' found {}'.format(self.iter_warmup)
+                    'Vaule for iter_warmup must be a non-negative integer,'
```

Comment:
```suggestion
                    'Value for iter_warmup must be a non-negative integer,'
```

Reply:
thanks! spell-checked the entire file - fixed several typos.